plugins/webp-uploads/settings.php

```diff
@@ -44,49 +55,53 @@ function webp_uploads_add_media_settings_field(): void {
 		is_multisite() ? 'default' : 'uploads',
 		array( 'class' => 'perflab-generate-webp-and-jpeg' )
 	);
+
+	// Add settings field.
+	add_settings_field(
+		'webp_uploads_use_picture_element',
+		__( 'Picture Element', 'webp-uploads' ),
+		'webp_uploads_use_picture_element_callback',
+		'media',
+		is_multisite() ? 'default' : 'uploads',
+		array( 'class' => 'webp-uploads-use-picture-element' )
+	);
 }
-add_action( 'admin_init', 'webp_uploads_add_media_settings_field' );
+add_action( 'admin_init', 'webp_uploads_add_media_settings_fields' );
 
 /**
  * Renders the settings field for the 'perflab_generate_webp_and_jpeg' setting.
  *
  * @since 1.0.0
  */
 function webp_uploads_generate_webp_jpeg_setting_callback(): void {
-	if ( ! is_multisite() ) {
-		?>
-			</td>
-			<td class="td-full">
-		<?php
-	}
+
 	?>
+	<tr><td colspan="2" class="td-full">
 		<label for="perflab_generate_webp_and_jpeg">
 			<input name="perflab_generate_webp_and_jpeg" type="checkbox" id="perflab_generate_webp_and_jpeg" aria-describedby="perflab_generate_webp_and_jpeg_description" value="1"<?php checked( '1', get_option( 'perflab_generate_webp_and_jpeg' ) ); ?> />
 			<?php esc_html_e( 'Generate JPEG files in addition to WebP', 'webp-uploads' ); ?>
 		</label>
 		<p class="description" id="perflab_generate_webp_and_jpeg_description"><?php esc_html_e( 'Enabling JPEG in addition to WebP can improve compatibility, but will effectively double the filesystem storage use of your images.', 'webp-uploads' ); ?></p>
+	</td></tr>
 	<?php
 }
 
 /**
- * Adds custom styles to hide specific elements in media settings.
+ * Renders the settings field for the 'webp_uploads_use_picture_element' setting.
  *
  * @since 1.0.0
  */
-function webp_uploads_media_setting_style(): void {
-	if ( is_multisite() ) {
-		return;
-	}
+function webp_uploads_use_picture_element_callback(): void {
 	?>
-	<style>
-		.form-table .perflab-generate-webp-and-jpeg th,
-		.form-table .perflab-generate-webp-and-jpeg td:not(.td-full) {
-			display: none;
-		}
-	</style>
+		<tr><td colspan="2" class="td-full">
+			<label for="webp_uploads_use_picture_element">
+			<input name="webp_uploads_use_picture_element" type="checkbox" id="webp_uploads_use_picture_element" aria-describedby="webp_uploads_use_picture_element_description" value="1"<?php checked( '1', get_option( 'webp_uploads_use_picture_element' ) ); ?> />
+			<?php esc_html_e( 'Use Picture Element', 'webp-uploads' ); ?>
```

Comment:
What do you think about using `<picture>` in the label>

![image](https://github.com/WordPress/performance/assets/134745/5a569020-f4ad-40eb-b950-00fb0b3f5fca)

Reply:
Yea, that does look better. Probably should change in most places we mention it.

Reply:
renamed in 5b687bcf9de8fad2a711fb2c291da2a7f3bb913e